tests/forte/data/data_store_test.py

```diff
@@ -0,0 +1,206 @@
+"""
+
+import logging
+import unittest
+from sortedcontainers import SortedList
+
+from forte.data.data_store import DataStore
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class DataStoreTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.data_store = DataStore()
+        # attribute fields for Document and Sentence entries
+        self.data_store._type_attributes = {
+            "Document": ["document_class", "sentiment", "classifications"],
+            "Sentence": [
+                "speaker",
+                "part_id",
+                "sentiment",
+                "classification",
+                "classifications",
+            ],
+        }
+        # The order is [Document, Sentence]. Initialize 2 entries in each list.
+        # Document entries have tid 1234, 3456.
+        # Sentence entries have tid 9999, 1234567.
+        # The type id for Document is 0, Sentence is 1.
+        self.data_store.elements = [
+            SortedList(
+                [
+                    ["Document", 1234, 0, 5, None, "Postive", None],
+                    [
+                        "Document",
+                        3456,
+                        10,
+                        25,
+                        "Doc class A",
+                        "Negative",
+                        "Class B",
+                    ],
+                ],
+                key=lambda x: (x[2], x[3]),
+            ),
+            SortedList(
+                [
+                    [
+                        "Sentence",
+                        9999,
+                        6,
+                        9,
+                        "teacher",
+                        1,
+                        "Postive",
+                        None,
+                        None,
+                    ],
+                    [
+                        "Sentence",
+                        1234567,
+                        55,
+                        70,
+                        None,
+                        None,
+                        "Negative",
+                        "Class C",
+                        "Class D",
+                    ],
+                ],
+                key=lambda x: (x[2], x[3]),
+            ),
+        ]
+        self.data_store.entry_dict = {
+            1234: ["Document", 1234, 0, 5, None, "Postive", None],
+            3456: [
+                "Document",
+                3456,
+                10,
+                25,
+                "Doc class A",
+                "Negative",
+                "Class B",
+            ],
+            9999: ["Sentence", 9999, 6, 9, "teacher", 1, "Postive", None, None],
+            1234567: [
+                "Sentence",
+                1234567,
+                55,
+                70,
+                None,
+                None,
+                "Negative",
+                "Class C",
+                "Class D",
+            ],
+        }
+
+    def test_add_annotation_raw(self):
```

Comment:
Simply implement one of the functions and test it here, you cannot test NontImpelemtned functions